ccd-definition/AuthorisationCaseField/court-admin.json

```diff
@@ -355,5 +355,12 @@
     "CaseFieldID": "temporaryC2Document",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "courtDirectionsCustom",
```

Comment:
added 
```
    "Comment": "courtDirectionsCustom is used here to prevent Gatekeeper inadvertently getting C and D permissions in draftSDO journey on courtDirections. We do not want them to able to create/delete directions"

```

to caseEventToFields